include/entity.hpp

```diff
@@ -32,7 +32,7 @@ class Entity {
 	 */
 	virtual bool collides(const Entity &) const { return false; };
 
-	virtual void update() { _pos += _velocity * glob::dt; };
+	virtual void update() { /*_pos += _velocity * glob::dt;*/ };
```

Comment:
Commented out for now. Let's write this out explicitly in the subclasses of `Entity` (such as `MainCharacter`) so that it's not unambiguous for the starters!

Reply:
Seems like a bad practice to me in general, but especially in this case. You have to explicitly call the superclass version of the method in order for this line to be called, so it's not like the call of this method is hidden in any way unless the type of entity doesn't override update at all. This reduces repeated code. Also, I corrected your phrasing :)